snow/consensus/snowman/metrics.go

```diff
@@ -218,10 +225,14 @@ func (m *metrics) Accepted(
 
 	m.pollsAccepted.Observe(float64(pollNumber - start.pollNumber))
 
-	duration := time.Since(start.time)
-	m.latAccepted.Observe(float64(duration))
+	now := time.Now()
+	processingDuration := now.Sub(start.time)
+	m.latAccepted.Observe(float64(processingDuration))
 
 	m.blockSizeAcceptedSum.Add(float64(blockSize))
```

Comment:
nit: move this before time metrics collected

Reply:
re-ordered the metrics